### PR TITLE
Homework #1:

### PR DESCRIPTION
**Pull request by testQuality100**
- TestCases: 5 test cases created for testing https://qa-complex-app-for-testing.herokuapp.com/
- Locators: locators added as acc to the used elements in TestCases

### CODE REVIEW

File: gnatyshkoProj/src/Locators [hunk 1]
```diff
@@ -1,4 +1,37 @@
 LoginPage
-button.login=".//button"
+input.username = ".//input[@id='username7']"
```
Comment (SeleniumQALight) [inline gnatyshkoProj/src/Locators:2]: Перевір чи спрацює цей локатор наступного разу . Пам'ятаєш про цифри ...

File: gnatyshkoProj/src/TestCases [hunk 2]
```diff
@@ -1,11 +1,57 @@
...
+The user sees that the post was created by his username and date of creation
+The user sees that the post was not written to all users
+The user sees the content of the post
+_______________________________________________________________________________________________________________________
+
+TC #4. Check the created post is present in MyProfile
+Prerequisite: The user is already registered in the website and is currently signed in. The PostPage is open in browser
+
+1. Click on 'My profile' sign
+2. In MyProfilePage in the list of posts click on the post with the title which matches with the title the post was earlier created
```
Comment (SeleniumQALight) [inline gnatyshkoProj/src/TestCases:44]: створення поста треба додати в прекондішени. Бо якщо базу почистимо - то не буде жодного поста.. А попередні кейси ми ж можем і не виконувать.

File: gnatyshkoProj/src/Locators (hunk 1 above)
Comment (testQuality100) [reply gnatyshkoProj/src/Locators:2]: дякую, виправила на 
input.username = ".//input[@name='username' and @placeholder='Username']"

File: gnatyshkoProj/src/TestCases (hunk 2 above)
Comment (testQuality100) [reply gnatyshkoProj/src/TestCases:44]: дякую, додала створення поста в prerequisite:

TC #4. Check the created post is present in MyProfile
Prerequisite: The user is already registered in the website and is currently signed in. The User has already created the post. The PostPage is open in browser

1. Click on 'My profile' sign
2. In MyProfilePage in the list of posts click on the post with the title which matches with the title the post was earlier created

**Review by SeleniumQALight — APPROVED** [review]

**Pull request merged**